source/game.programme.programmelicence.bmx

```diff
@@ -2244,6 +2245,9 @@ Type TProgrammeLicence Extends TBroadcastMaterialSource {_exposeToLua="selected"
 		endif
 		If HasBroadcastLimit() then showMsgBroadcastLimit = True
 		If HasBroadcastTimeSlot() then showMsgBroadcastTimeSlot = True
+		'TODO maybe disable showMsgPlannedWarning if in this case?
+		'planned is more interesting for single episodes
+		If IsSeries() And IsCustomProduction() And Not IsTradeable() then showMsgInProduction = True
```

Comment:
dem Datenblattanzeiger `Method ShowProgrammeSheet:Int(x:Int,y:Int, align:Float=0.5, useOwner:int=-1, extraData:TData = null)` koennte man noch einen Parameter spendieren - der den Raum uebergibt (aehnlich den "Lizenz/Werbevertragslisten"). Dann kann diese Meldung im Programmplaner "entfallen".

Ob man nun den Aufwand aber betreiben muss...

Reply:
Es ging mir eher darum, zu verhindern, dass zu viele Nachrichten angezeigt (Serie von Live-Veranstaltungen mit Ausstrahlungslimit und Begrenzung der Ausstrahlungszeit) werden und die Information "im Programmplan" ist für den Serienheader doch eher uninteressant, oder?

Reply:
Da hast Du recht ... ich wollte es nur generell noch weiter einschraenken, aber das "IsTradeable()" sollte schon ausreichen.

Etwas unschoen ist das fixieren auf "IsTradeable()" vielleicht doch eine "IsUnfinishedCustomProduction()" einfuehren die dann `Return IsCustomProduction() and not IsTradeable()` zurueckgibt und im Fall der Notwendigkeit dann spaeter mit einer "passenderen" Logik ausgestattet werden kann.

Reply:
Bezüglich isUnfinishedCustomProduction kann ich mir die Änderungen nochmal anschauen - von der API her wäre das sauberer. Natürlich unterscheidet sich die Berechnung für Film/Live-Programm und Serie.